airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -585,26 +593,16 @@ def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, Any]]]:
             for branch in self.branches_to_pull.get(repository, []):
                 yield {"branch": branch, "repository": repository}
 
-    def parse_response(
-        self,
-        response: requests.Response,
-        stream_state: Mapping[str, Any],
-        stream_slice: Mapping[str, Any] = None,
-        next_page_token: Mapping[str, Any] = None,
-    ) -> Iterable[Mapping]:
-        for record in response.json():  # GitHub puts records in an array.
-            yield self.transform(record=record, repository=stream_slice["repository"], branch=stream_slice["branch"])
-
-    def transform(self, record: MutableMapping[str, Any], repository: str = None, branch: str = None, **kwargs) -> MutableMapping[str, Any]:
-        record = super().transform(record=record, repository=repository)
+    def transform(self, record: MutableMapping[str, Any], stream_slice: Mapping[str, Any]) -> MutableMapping[str, Any]:
+        record = super().transform(record=record, stream_slice=stream_slice)
 
         # Record of the `commits` stream doesn't have an updated_at/created_at field at the top level (so we could
         # just write `record["updated_at"]` or `record["created_at"]`). Instead each record has such value in
         # `commit.author.date`. So the easiest way is to just enrich the record returned from API with top level
         # field `created_at` and use it as cursor_field.
         # Include the branch in the record
         record["created_at"] = record["commit"]["author"]["date"]
```

Comment:
What if `record["commit"]["author"]["date"]` is missing? Can we use the `.get()` instead?

Reply:
in really I don't know
it's logic which I did not touched in scope of this PR